bundles/org.openhab.binding.cul.max/README.md

```diff
@@ -0,0 +1,204 @@
+using the [CUL USB dongle](http://busware.de/tiki-index.php?page=CUL) rather than the MAX!Cube.
+This should allow greater control over the devices than the cube offers as all interaction is handled manually.
+
+## Supported Things
+
+This binding support 6 different things types
+
+| Thing          | Type   | Description                                                                                                        |
+|----------------|--------|--------------------------------------------------------------------------------------------------------------------|
+| cul_max_bridge | Bridge | CUL if using a serial CUL
+| cun_max_bridge | Bridge | CUN if using a network CUL
+| maxcul_bridge  | Bridge | This is the MAX! Cube emulation for the CUL.                                                                                 |
+| thermostat     | Thing  | This is for the MAX! Heating Thermostat. This is also used for the power plug switch "Zwischenstecker-Schaltaktor". |
+| thermostatplus | Thing  | This is for the MAX! Heating Thermostat+. This is the type that can hold the program by itself.                    |
+| wallthermostat | Thing  | MAX! Wall Thermostat.                                                                                              |
+| ecoswitch      | Thing  | MAX! Ecoswitch.                                                                                                    |
+| shuttercontact | Thing  | MAX! Shuttercontact / Window Contact.                                                                              |
+
+Generally one does not have to worry about the thing types as they are automatically defined.
+If for any reason you need to manually define the Things and you are not exactly sure what type of thermostat you have, you can choose `thermostat` for both the thermostat and thermostat+, this will not affect their working.
+
+## Discovery
+
+Start pairing at your MAX! device. It will show up in the inbox and you can it to your configuration. Serial number and rfAddress is set already
+After adding too your configuration you have to set channel pair_mode to ON on your maxcul_bridge and restart
+the pairing on your MAX! device to start the pairing sequence. 
+If the device is already paired and starts re-pairing e.g. after battery change, the state of channel pair_mode is ignored.
+
+## Binding Configuration
+
+There are no binding wide settings as all configuration settings.
+When using a serial port, you may need to add `-Dgnu.io.rxtx.SerialPorts=/dev/ttyACM0` in your server startup.  Please consult the [forum](https://community.openhab.org) for the latest information.
+
+## Thing Configuration
+
+### CUL Bridge Configuration
+| Property | Default | Required | Description |
+|----------|---------|:--------:|-------------|
+| device   |         |   Yes    | in the form `<device>`, where `<device>` is a local serial port eg `/dev/ttyACM0` |

```

Comment:
This seems a bit confusing. The parameter value should only by the serial port? If so, better remove the `<device>` reference.

Reply:
In OH1 it was a global configuration for serial and network for the whole binding and I copied the documentation but it doesn't make sense anymore. I will change it.